pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -742,30 +741,20 @@ private String extractStorageLocation(ResultSet rs) throws SQLException {
      */
     public enum AttributeKey {
 
-      SAMPLE_CATEGORY("sample_category", "Sample Category"),
-      RECEIVE_DATE("receive_date", "Receive Date") {
+      SAMPLE_CATEGORY("sample_category", "Sample Category"), RECEIVE_DATE("receive_date", "Receive Date") {
         @Override
         public String extractStringValueFrom(ResultSet rs) throws SQLException {
           return rs.getDate(getSqlKey()) == null ? null : rs.getDate(getSqlKey()).toString();
         }
       },
-      EXTERNAL_NAME("external_name", "External Name"),
-      SEX("sex", "Sex") {
+      EXTERNAL_NAME("external_name", "External Name"), SEX("sex", "Sex") {
         @Override
         public String extractStringValueFrom(ResultSet rs) throws SQLException {
           String str = rs.getString(getSqlKey());
           return WordUtils.capitalizeFully(str);
         }
       },
-      TISSUE_ORIGIN("tissue_origin", "Tissue Origin"),
-      TISSUE_TYPE("tissueType", "Tissue Type"),
-      TISSUE_PREPARATION("tissue_preparation", "Tissue Preparation"),
-      TISSUE_REGION("tissue_region", "Region"),
-      TUBE_ID("tube_id", "Tube Id"),
-      GROUP_ID("group_id", "Group ID"),
-      GROUP_DESCRIPTION("group_id_description", "Group Description"),
-      ORGANISM("organism", "Organism"),
-      PURPOSE("purpose", "Purpose") {
+      TISSUE_ORIGIN("tissue_origin", "Tissue Origin"), TISSUE_TYPE("tissueType", "Tissue Type"), TISSUE_PREPARATION("tissue_preparation", "Tissue Preparation"), TISSUE_REGION("tissue_region", "Region"), TUBE_ID("tube_id", "Tube Id"), GROUP_ID("group_id", "Group ID"), GROUP_DESCRIPTION("group_id_description", "Group Description"), ORGANISM("organism", "Organism"), PURPOSE("purpose", "Purpose") {
```

Comment:
:crying_cat_face: looks like it needs terminal `//`s

Reply:
In Eclipse project settings > Java Code Style > Formatter, "Enable project-specific settings" should be enabled. If you edit the active profile, Line Wrapping > "Never join already wrapped lines" should be selected, and that should prevent this from happening. These settings should be saved in the repository. What part of it isn't working?

Reply:
Fixed. The project-specific settings weren't enabled. :cry: